docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -1,94 +1,93 @@
-# Shipping Providers
+# Become a Shipping Provider
 
-Shipping service providers wishing to offer shipping services and rates to BigCommerce merchants and shoppers can implement BigCommerce Shipping Provider endpoints. Once they implement and accept their service into BigCommerce's shipping carrier registry, merchants will have access to enable and configure the service through their BigCommerce control panel. Once enabled on a store, BigCommerce will automatically retrieve the service options and rates using the provider's endpoints and display them to merchants in the store's control panel and to shoppers on the storefront.
+Shipping service providers who wish to offer shipping services and rates to BigCommerce merchants and shoppers can implement endpoints through BigCommerce. Once they implement the endpoints and BigCommerce registers their shipping provider app, a merchant designated as store owner can install their app on the store. Merchants and API users can then connect and enable the service on their store via the Shipping Manager UI and the [Shipping V2 API](/api-reference/store-management/shipping-api), respectively. BigCommerce will then automatically retrieve the service options and rates using the provider's endpoints, displaying them to merchants in the store's control panel and to shoppers on the storefront. Merchants and API users can define and enable a real-time shipping method for your carrier app in one or more shipping zones. They may define and enable a different shipping method in each shipping zone for your carrier. 
 
-Merchants can also use shipping provider endpoints to retrieve rates from custom shipping tables or an in-house shipping rate calculation service.
+The Shipping Provider API provides many benefits:
 
-Some use cases for the Shipping Provider API are:
+- Drop-shippers can require their own rates
+- Merchants can retrieve rates from custom shipping tables or in-house shipping rate calculation services 
+- Shoppers can create a combination of in-store pickup and shipping options
 
-- A drop-shipper that requires their own rates
-- A merchant that already has a shipping table
-- Third-party logistics
-- Create a combination of in-store pickup and shipping options for shoppers
+This article guides developers on how to create and register a BigCommerce shipping provider app that will make your shipping rates available to merchants and shoppers on demand.

```

Comment:
you're shifting between second and third person here. our style guide permits both. I don't really care about you picking one to use for the entire article, but you should stay consistent within a single sentence.